west.yml

```diff
@@ -42,7 +42,7 @@ manifest:
       groups:
         - tools
     - name: fatfs
-      revision: 94fcd6bfb3801ac0a5e12ea2f52187e0a688b90e
+      revision: /pull/9/head
```

Comment:
Sorry, my fault: should be `pull/9/head`, without leading `/`.

Reply:
Hi @de-nordic 

Ah okay, no issue :)  force pushed with the changes without leading / .

Regards, 
Boon Khai